code/game/machinery/floodlight.dm

```diff
@@ -59,7 +61,7 @@
 			return
 		on = 1
 		user << "\blue You turn on the light"
-		set_light(brightness_on)
+		set_light(brightness_range, brightness_power, light_color)
```

Comment:
The issue with floodlights is their light color, not the range of the light. It was set to look whiter than normal light originally, but when we added the new light system, it became very dim. Removing the light color should fix this problem.